src/func/alias.js

```diff
@@ -105,3 +105,58 @@ export async function getAliasByIds(
 	return aliases.reduce((aliasesMap, alias) =>
 		_.assign(aliasesMap, {[alias.id]: snakeToCamel(alias)}), {});
 }
+
+export function getAliasIds(
+	transacting: Transaction, name: string, caseSensitive: boolean = false
+): Promise<Array<Object>> {
+	if (caseSensitive) {
+		return transacting.select('id')
+			.from('bookbrainz.alias')
+			.where('name', name);
+	}
+	return transacting.select('id').from('bookbrainz.alias').where(
+		transacting.raw('LOWER("name") = ?', name.toLowerCase())
+	);
+}
+
+export async function getEntitiesWithMatchingAlias(
```

Comment:
I would rename getEntitiesWithMatchingAlias to getBBIDsMatchingAlias or getBBIDsForAlias, since the code only returns bbids.
Change to be replicated at line 158 below and in /bookbrainz-site/src/server/helpers/search.js line 346 (on pr/195 and pr/208 branches)

Reply:
Other than that, good work.